common.mk

```diff
@@ -46,9 +46,9 @@ LD_FLAGS = -s -w
 GC_FLAGS =
 endif
 
-LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Date=$(BUILD_DATE)'
-LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.SHA=$(BUILD_SHA)'
-LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.Version=$(BUILD_VERSION)'
+LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo.Date=$(BUILD_DATE)'
+LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo.SHA=$(BUILD_SHA)'
+LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo.Version=$(BUILD_VERSION)'
```

Comment:
I think it would be better to not touch this file and keep using existing builinfo in this PR until Harish PR get merged (anyways for the ```discovery``` package, there is dependency on TF until that PR get merged. can always rebase after that)

Reply:
Good idea. I"ll go back to the old buildinfo path.